### PR TITLE
Update ruby 2.5 on travis runners

### PR DESCRIPTION
**Pull request by ngetahun**
## Description

Please describe your change and provide as much context as possible.

Fixes # (issue)

## Change Type

*Please select the correct option.*

- [x] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [x] I have documented the `MANUAL.md` file with any changes to the user experience.
- [x] RMT's test coverage remains at 100%.
- [x] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
   # SLES15
-  - 2.5.0
+  - 2.5.8
```
Comment (digitaltom) [inline .travis.yml:4]: I suggest to use SCC package search to find all Ruby versions that we need to support and add them here.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/193907/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged a23b3580bb88d66f35bcaaa6a0a96504ef181bc1 into 71386fd62b5658eb00ae2881b8a9b8b12ae1e83c)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/193919/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 7560a7d03ac6944d2114e407a4e93c615ee0f231 into 71386fd62b5658eb00ae2881b8a9b8b12ae1e83c)

File: .travis.yml (hunk 1 above)
Comment (Sergeykot) [reply .travis.yml:4]: Screenshot_20210518_154822

Not sure if we should do it like that, if I search in package search for Ruby package we have all versions there
maybe is enough to have latest minor versions only, which PR does?

**Review by Sergeykot — APPROVED** [review]
LGTM :+1:

File: .travis.yml (hunk 1 above)
Comment (ngetahun) [reply .travis.yml:4]: I added the latest minor version of Ruby 2.5+

**Pull request merged**